hermes/templates/hermes/posts_for_tag.html

```diff
@@ -11,7 +11,7 @@ <h2>On the blog</h2>
   {% endthumbnail %}
   {% endif %}
   <div class="ds-card-body">
-    <h3><a class="ds-card-link ds-h5" href="#">{{ post.subject }}</a></h3>
+    <h3><a class="ds-card-link ds-h5" href="#{{ post.get_absolute_url }}">{{ post.subject }}</a></h3>
```

Comment:
Dont think the `#` should be included should it, as this looks like we are linking to another page rather than an anchor tag on this page?

Reply:
Nope, I fixed it on another branch; dropping it from this PR now.